<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FfmpegVideoToolClass</class>
 <widget class="QMainWindow" name="FfmpegVideoToolClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>949</width>
    <height>831</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FfmpegVideoTool</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableWidget" name="VideoTableWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>531</width>
      <height>351</height>
     </rect>
    </property>
   </widget>
   <widget class="QOpenGLWidget" name="openGLWidget">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>50</y>
      <width>391</width>
      <height>291</height>
     </rect>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>510</y>
      <width>911</width>
      <height>171</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Agency FB&quot;;
QHeaderView::section{ background-color:rgb(50,143,218);
font:13pt'宋体';
color: white;};</string>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>信息</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>转码</string>
     </attribute>
     <widget class="QComboBox" name="comboBoxFormat">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>20</y>
        <width>171</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="OutFormat">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;Arial&quot;;</string>
      </property>
      <property name="text">
       <string>输出格式</string>
      </property>
     </widget>
     <widget class="QLabel" name="VideoQuality">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>55</y>
        <width>72</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;Arial&quot;;</string>
      </property>
      <property name="text">
       <string>视频质量</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxQuality">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>50</y>
        <width>171</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="UploadURL">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>72</width>
        <height>15</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;Arial&quot;;</string>
      </property>
      <property name="text">
       <string>上传URL</string>
      </property>
     </widget>
     <widget class="QPushButton" name="oneClickProBtn">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>30</y>
        <width>93</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string>开始转码</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>20</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;Arial&quot;;</string>
      </property>
      <property name="text">
       <string>分辨率</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxSize">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>20</y>
        <width>201</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>90</y>
        <width>291</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>合并</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>截取</string>
     </attribute>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>时间段</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="timePointEdit">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>20</y>
        <width>221</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="selTimeBtn">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>20</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>选择</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>帧操作</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QLabel" name="OutDir">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>740</y>
      <width>72</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>输出目录</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="outDirEditForTran">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>740</y>
      <width>291</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="selectOutDirBtn">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>740</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startVideoProBtn">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>700</y>
      <width>93</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>开始执行</string>
    </property>
   </widget>
   <widget class="QWidget" name="extractSetWidget" native="true">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>0</y>
      <width>701</width>
      <height>571</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(220, 220, 220)</string>
    </property>
    <widget class="QPushButton" name="saveAndExitBtn">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>510</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
    <widget class="QPushButton" name="unsaveAndExitBtn">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>510</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
    <widget class="QLabel" name="picLabel">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>641</width>
       <height>361</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="startTimeLabel">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>390</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>00:00:00</string>
     </property>
    </widget>
    <widget class="QSlider" name="playCtrlSlider">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>430</y>
       <width>631</width>
       <height>22</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="FfmpegVideoTool.qrc"/>
 </resources>
 <connections/>
</ui>
